modern/src/DevicePage.js

```diff
@@ -71,22 +76,47 @@ const DevicePage = () => {
       <Container maxWidth='xs' className={classes.container}>
         <form>
           {(!id || device) &&
-            <TextField
-              margin='normal'
-              fullWidth
-              defaultValue={device && device.name}
-              onChange={(event) => setName(event.target.value)}
-              label={t('sharedName')}
-              variant='filled' />
-          }
-          {(!id || device) &&
-            <TextField
-              margin='normal'
-              fullWidth
-              defaultValue={device && device.uniqueId}
-              onChange={(event) => setUniqueId(event.target.value)}
-              label={t('deviceIdentifier')}
-              variant='filled' />
+            <div>
```

Comment:
Is there a way to avoid `div`? Can we use fragment here?

Reply:
Sorry I was unaware of fragment.